test/csuite/Makefile.am

```diff
@@ -167,6 +167,10 @@ test_wt8246_compact_rts_data_correctness_SOURCES = wt8246_compact_rts_data_corre
 noinst_PROGRAMS += test_wt8246_compact_rts_data_correctness
 all_TESTS += test_wt8246_compact_rts_data_correctness
 
+test_wt8659_reconstruct_database_from_logs_SOURCES = wt8659_reconstruct_database_from_logs/main.c
```

Comment:
Please sync with `develop`, this file no longer exists.

Reply:
I'll do that last thing before merging into develop.